rackcity/views/decommission_views.py

```diff
@@ -267,12 +269,14 @@ def decommissioned_asset_page_count(request):
     Return total number of pages according to page size, which must be
     specified as query parameter.
     """
-    (change_plan, failure_response) = get_change_plan(query_params.get("change_plan"))
+    (change_plan, failure_response) = get_change_plan(
+        request.query_params.get("change_plan")
+    )
```

Comment:
Was this just a bug that we weren't calling `request.query_params`? Did we know about that?

Reply:
yeah this was an uncaught bug. not sure when it got introduced. we could look at the git blame for fun